index.js

```diff
@@ -139,14 +144,17 @@ Nanobus.prototype._emit = function (arr, eventName, data, uuid) {
     data = eventName
     eventName = null
   }
+
   var length = arr.length
+  var args = data
```

Comment:
that line is how the default use case of `listener.apply(listener, data)` gets set up. Only in the case where `eventName` is set, does it alter the list of arguments.